src/test/java/uk/gov/companieshouse/overseasentitiesapi/utils/NaturesOfControlCollectionBuilderTest.java

```diff
@@ -0,0 +1,53 @@
+class NaturesOfControlCollectionBuilderTest {
+
+
+    @Test
+    void testThereAreNoDuplicateBuilds() {
+        var firstNaturesOfControlCollection =
+                NaturesOfControlCollectionBuilder.createNaturesOfControlCollectionBuilder()
+                        .addPersonType(Arrays.asList(NatureOfControlType.SIGNIFICANT_INFLUENCE_OR_CONTROL))
+                        .addTrusteesType(Arrays.asList(NatureOfControlType.OVER_25_PERCENT_OF_SHARES))
+                        .addFirmType(Arrays.asList(NatureOfControlType.OVER_25_PERCENT_OF_VOTING_RIGHTS))
+                        .addTrustType(Arrays.asList(NatureOfControlType.SIGNIFICANT_INFLUENCE_OR_CONTROL))
+                        .addOwnerOfLandPerson(Arrays.asList(NatureOfControlJurisdictionType.SCOTLAND))
+                        .addOwnerOfLandOtherEntity(Arrays.asList(NatureOfControlJurisdictionType.ENGLAND_AND_WALES))
+                        .addFirmControlType(Arrays.asList(NatureOfControlType.OVER_25_PERCENT_OF_SHARES))
+                        .addFeatureFlag(true)
+                        .build();
+        assertNotNull(firstNaturesOfControlCollection.getPersonNatureOfControlTypes());
+        assertNotNull(firstNaturesOfControlCollection.getTrusteesNatureOfControlTypes());
+        assertNotNull(firstNaturesOfControlCollection.getFirmNatureOfControlTypes());
+        assertNotNull(firstNaturesOfControlCollection.getTrustControlNatureOfControlTypes());
+        assertNotNull(firstNaturesOfControlCollection.getOwnerOfLandPersonNatureOfControlJurisdictions());
+        assertNotNull(firstNaturesOfControlCollection.getOwnerOfLandOtherEntityNatureOfControlJurisdictions());
+        assertNotNull(firstNaturesOfControlCollection.getFirmControlNatureOfControlTypes());
+
+        var secondNaturesOfControlCollection =
+                NaturesOfControlCollectionBuilder.createNaturesOfControlCollectionBuilder()
+                        .addPersonType(Arrays.asList(NatureOfControlType.SIGNIFICANT_INFLUENCE_OR_CONTROL))
+                        .addTrusteesType(Arrays.asList(NatureOfControlType.OVER_25_PERCENT_OF_SHARES))
+                        .addFirmType(Arrays.asList(NatureOfControlType.OVER_25_PERCENT_OF_VOTING_RIGHTS))
+                        .addFeatureFlag(true)
+                        .build();
+
+        assertNotNull(secondNaturesOfControlCollection.getPersonNatureOfControlTypes());
+        assertNotNull(secondNaturesOfControlCollection.getTrusteesNatureOfControlTypes());
+        assertNotNull(secondNaturesOfControlCollection.getFirmNatureOfControlTypes());
+        assertNull(secondNaturesOfControlCollection.getTrustControlNatureOfControlTypes());
+        assertNull(secondNaturesOfControlCollection.getOwnerOfLandPersonNatureOfControlJurisdictions());
+        assertNull(secondNaturesOfControlCollection.getOwnerOfLandOtherEntityNatureOfControlJurisdictions());
+        assertNull(secondNaturesOfControlCollection.getFirmControlNatureOfControlTypes());
+
```

Comment:
Since I'm being pedantic and commenting on blank lines, one of these should go.

Reply:
Removed